gen/shuffleboard/ShuffleboardContainer.yml

```diff
@@ -14,64 +17,148 @@ classes:
           std::string_view, int:
+            return_value_policy: reference_internal
           std::string_view, std::string_view:
+            return_value_policy: reference_internal
           std::string_view, char*:
             ignore: true
           std::string_view, std::span<const bool>:
+            return_value_policy: reference_internal
           std::string_view, std::span<const double>:
+            return_value_policy: reference_internal
           std::string_view, std::span<const float>:
+            return_value_policy: reference_internal
           std::string_view, std::span<const int64_t>:
+            return_value_policy: reference_internal
           std::string_view, std::span<const std::string>:
+            return_value_policy: reference_internal
       AddCamera:
-        ignore: true
+        ignore: true # AddCamera is broken when DYNAMIC_CAMERA_SERVER is defined
       AddString:
+        return_value_policy: reference_internal
       AddNumber:
+        return_value_policy: reference_internal
       AddDouble:
+        return_value_policy: reference_internal
       AddFloat:
+        return_value_policy: reference_internal
       AddInteger:
+        return_value_policy: reference_internal
       AddBoolean:
+        return_value_policy: reference_internal
       AddStringArray:
+        return_value_policy: reference_internal
       AddNumberArray:
+        return_value_policy: reference_internal
       AddDoubleArray:
+        return_value_policy: reference_internal
       AddFloatArray:
+        return_value_policy: reference_internal
       AddIntegerArray:
+        return_value_policy: reference_internal
       AddBooleanArray:
+        return_value_policy: reference_internal
       AddRaw:
         overloads:
           std::string_view, std::function<std::vector<uint8_t> ( )>:
+            return_value_policy: reference_internal
           std::string_view, std::string_view, std::function<std::vector<uint8_t> ( )>:
+            return_value_policy: reference_internal
       AddPersistent:
         overloads:
           std::string_view, nt::Value&:
+            return_value_policy: reference_internal
           std::string_view, bool:
+            return_value_policy: reference_internal
           std::string_view, double:
+            return_value_policy: reference_internal
           std::string_view, float:
+            return_value_policy: reference_internal
           std::string_view, int:
+            return_value_policy: reference_internal
           std::string_view, std::string_view:
+            return_value_policy: reference_internal
           std::string_view, std::span<const bool>:
+            return_value_policy: reference_internal
           std::string_view, std::span<const double>:
+            return_value_policy: reference_internal
           std::string_view, std::span<const float>:
+            return_value_policy: reference_internal
```

Comment:
Will this overload be reachable? Maybe passing in a numpy float32 array could work (I haven't tried), but will it hit the `std::span<const double>` type caster before reaching this?

Reply:
Probably not. But really, will anyone care?

Reply:
It'll probably be confusing if we leave an unreachable thing documented.

Reply:
Well when someone tries to reach it they can file a bug, and then we know someone cares about it.